test/e2e/specs/editor/various/navigable-toolbar.spec.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Block Toolbar', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test.describe( 'Contextual Toolbar', () => {
+		test( 'should not scroll page', async ( { page, pageUtils } ) => {
+			while (
+				await page.evaluate( () => {
+					const scrollable = window.wp.dom.getScrollContainer(
+						document.activeElement
+					);
+					return ! scrollable || scrollable.scrollTop === 0;
+				} )
+			) {
+				await page.keyboard.press( 'Enter' );
+			}
```

Comment:
I'm not sure why yet, but when I run this test locally and watch it in the browser, it gets stuck in a cycle of pressing Enter over and over without moving on to the code after it.

Reply:
I think something got messed up during a copy/paste or search/replace. I copy/pasted the old test from `[navigable-toolbar.test](https://raw.githubusercontent.com/WordPress/gutenberg/6c0f4a50324b7f09b4c13c8ce2b974b1e2243bd1/packages/e2e-tests/specs/editor/various/navigable-toolbar.test.js)` and replaced the `expect( await isInBlockToolbar() ).toBe( true );` part, and the test passes. 

Could you try copy/pasting it again and see if it works for you?